modules/social_features/social_core/config/update/social_core_update_8809.yml

```diff
@@ -0,0 +1,10 @@
+block.block.socialblue_local_actions:
+  expected_config:
+    visibility:
+      request_path:
+        pages: "/admin/reports/*\r\n/group/*/membership\r\n/node/*/all-enrollments"
```

Comment:
I'm wondering whether we should just remove this line altogether in case someone has already manually adjusted the block for some pages. But I'm not sure.

Reply:
That would be easier for us, but for BC with other developers, I think it is safer just add more pages here.

Reply:
Well, I mostly mean that in case someone has already done something like set the pages to 

```
"/admin/reports/*\r\n/group/*/membership\r\n/node/*/all-enrollments\r\n/node/*/delete"
```
or even
```
"/admin/reports/*\n/group/*/membership\n/node/*/all-enrollments"
```

Then the update hook as-is will fail.